pubsub/google/cloud/pubsub_v1/subscriber/policy/base.py

```diff
@@ -291,7 +291,7 @@ def lease(self, ack_id, byte_size):
         # If we do, we need to stop the stream.
         if self._load >= 1.0:
             self._paused = True
-            self.close()
+            self._consumer.pause()
```

Comment:
@lukesneeringer Should this also halt the thread running `maintain_leases`?

The whole point of `pause` is that we want to stop our `QueueCallbackWorker` from receiving new "dispatch" instructions because it will convert those into more requests to send (and we need to clear the unsent).

Reply:
No, it is important that leases continue to be maintained.